.gitignore

```diff
@@ -10,3 +10,4 @@ coverage/
 *.log
 *.debug
 gulpTODO.md
+.ranvierSecret
```

Comment:
Sure. In this case I just used a random word but I'd assume they'd want some kind of randomly generated text in a file that could be shared between server and client. I'll write a thing about it later today.